src/js/edu-benefits/5490/config/form.js

```diff
@@ -320,6 +336,7 @@ const formConfig = {
               'ui:options': {
                 expandUnder: 'view:applicantServed'
               },
+              'ui:required': _.get('view:applicantServed'),
```

Comment:
Ohhh, I see. Because it would "normally" be something like:
```js
(formData) => _.get('view:applicantServed', formData)
```
but because of the auto-currying, `_.get()` returns a function which `formData` is then passed into. It's tricky and I like it, but it's...not terribly clear, methinks.

On the other hand, what else would it be doing? I can see us going either way.